client/src/components/SideDrawer/index.tsx

```diff
@@ -0,0 +1,211 @@
+  //       <ListItemButton sx={{ pl: 4 }} onClick={() => navigate('../', { replace: true })}>
+  //         <ListItemText primary={item.startDate} />
+  //       </ListItemButton>
+  //     </List>
+  //   )
+  // }
+
+  // let list = itemsData && itemsData.map((item: any) => {return buildList(item)})
+
+  return (
+    <Box sx={{ display: 'flex' }}>
+      <AppBar position="fixed" open={open}>
+        <Toolbar>
+          <IconButton
+            color="inherit"
+            aria-label="open drawer"
+            onClick={handleDrawerOpen}
+            edge="start"
+            sx={{ mr: 2, ...(open && { display: 'none' }) }}
+          >
+            <MenuIcon />
+          </IconButton>
+          <Typography variant="h6" noWrap component="div">
+            NextRound
+          </Typography>
+        </Toolbar>
+      </AppBar>
+      <Drawer
+        sx={{
+          width: drawerWidth,
+          flexShrink: 0,
+          '& .MuiDrawer-paper': {
+            width: drawerWidth,
+            boxSizing: 'border-box',
+          },
+        }}
+        variant="persistent"
+        anchor="left"
+        open={open}
+      >
+        <DrawerHeader>
+          <IconButton onClick={handleDrawerClose}>
+            {theme.direction === 'ltr' ? <ChevronLeftIcon /> : <ChevronRightIcon />}
+          </IconButton>
+        </DrawerHeader>
+        <Divider />
+        <List>
+          {/* TODO change hrefs */}
+          <Box textAlign='center'>
+            <Button variant="contained" color="primary" style={{width: '90%'}} onClick={() => navigate('../', { replace: true })}>Add Job Cycle</Button> 
+          </Box>
+          <ListItem button key="Home" onClick={() => navigate('../', { replace: true })}>
+            <ListItemIcon> <HomeIcon /> </ListItemIcon>
+            <ListItemText primary="Home" />
+          </ListItem>
+          <ListItem button key="Metrics" onClick={() => navigate('../', { replace: true })}>
+              <ListItemIcon> <BarChartIcon /> </ListItemIcon>
+              <ListItemText primary="Metrics" />
+            </ListItem>
+            <ListItem button key="Forum" onClick={() => navigate('../', { replace: true })}>
+              <ListItemIcon> <ForumIcon /> </ListItemIcon>
+              <ListItemText primary="Forum" />
+            </ListItem>
+            <ListItem button key="Settings" onClick={() => navigate('../', { replace: true })}>
+              <ListItemIcon> <SettingsIcon /> </ListItemIcon>
+              <ListItemText primary="Settings" />
+            </ListItem>
+            <ListItem button key="Archives" onClick={handleClick}>
+              <ListItemIcon> <InventoryIcon /> </ListItemIcon>
+              <ListItemText primary="Archives" />
+              {ddOpen ? <ExpandLess /> : <ExpandMore />}
+            </ListItem>
+            <Collapse in={ddOpen} timeout="auto" unmountOnExit>
+              {/* {list} */}
+              <List component="div" disablePadding>
```

Comment:
More of a TODO than something to change. This will obviously be a map of the archived cycles for the users that each component will be built from. Resolve this comment after reading it; you can add the TODO comment in your code if it helps.

Reply:
I write some initial code for the {list} which maps over all cycles for a user